### ISSUE TITLE
[Feature] Switch between light and dark theme

### ISSUE THREAD

**Issue by Pazns** [FreshRSS/FreshRSS#1527]
This is a feature proposal.

_:: Use case ::_ 
I use FreshRSS daily in three situations.
- at home, in a dark environment ;
- at work, in a very bright and open one ;
- in underground transport, in a dark one.

For the sake of contrast and brightness, I need to switch regularly between a bright and a dark theme.
Today this is not very user-friendly, especially on mobile.

_:: Possible feature ::_ 
The user can switch between a dark and a light theme to accommodate his environment, without reloading the page and memorized by client.

_:: Possible definition ::_  
To keep current functionalities unharmed, like custom themes, and have a smooth evolution, a basic two-way switch could be enough.
It allows the user to alternate between two user-selected theme defined in the setting screen.
Selection is stored in client space, with cookies or local webstorage.
The first theme is default if no user selection found.
Both themes are loaded by the browser allowing real-time switch without reloading, useful for users with a bad network situation (subway, poor roaming in dense areas).

**Comment by aledeg** [comment]
I can see 2 different paths to achieve that:

1. upgrade themes to support (or not) a dark version of it. This way you have some consistencies when you switch from dark to light. This toggle action will be available only for theme with 2 version of it.
1. display the list of themes directly on the interface and select the one you want to use.

There might be other paths. If so feel free to expand the list.

ATM, I prefer the first solution. It involves more work but I think will be better in the long run.

**Comment by Pazns** [comment]
@aledeg 
Indeed, your observation n°2 is much more efficient and simple than what I saw first.

**Comment by Alkarex** [comment]
I think the list of themes in the main UI would take too much room. But maybe a single button to switch to the next theme, and/or a keyboard shortcut (what about mobile?) could do the job?

**Comment by Alkarex** [comment]
Related to https://github.com/FreshRSS/FreshRSS/issues/264

**Comment by Grishkaone** [comment]
Hello ! Is there any news on this subject ? 
I found myself regularly switching between a dark and a light theme, but it's quite laborious to use the setting menu each time.

I can see to place where it can be : a simple button in the main interface, or a button in the main menu. Not a new line, but a single button on the right side of the "affichage" line, like this :
![image](https://user-images.githubusercontent.com/46840243/147941800-5cc4b4bf-cbb2-48c2-9318-7434cbcdccf9.png)

No wasted space.

**Comment by Pazns** [comment]
Since then, I ended up using [Dark Reader](https://darkreader.org/) browser add-on everywhere, which renders this feature irrelevant for me.

I also think now that a proper FreshRSS theme should include internally the light/dark theme, with the now widely supported CSS feature "prefers-color-scheme".

It would gives much more usability than a manual-only switch.

**Comment by math-GH** [comment]
#4515

**Comment by Alkarex** [comment]
https://github.com/FreshRSS/Extensions/pull/140
